10-dns-bootstrap.md

```diff
@@ -1,68 +1,136 @@
-The DNS server is authoritative for a subdomain, called a _seed root domain_, and clients may query it for subdomains.
+A domain name server implementing this specification is referred to as a
+_DNS Seed_ and answers incoming DNS queries of type `A`, `AAAA`, or `SRV`, as
+specified in RFCs 1035<sup>[1](#ref-1)</sup>, 3596<sup>[2](#ref-2)</sup>, and
+2782<sup>[3](#ref-3)</sup>, respectively.
+The DNS server is authoritative for a subdomain, referred to as a
+_seed root domain_, and clients may query it for subdomains.
 
 The subdomains consist of a number of dot-separated _conditions_ that further narrow the desired results.
 
+## Table of Contents
+
+  * [DNS Seed Queries](#dns-seed-queries)
+    * [Query Semantics](#query-semantics)
+  * [Reply Construction](#reply-construction)
+  * [Policies](#policies)
+  * [Examples](#examples)
+  * [References](#references)
+  * [Authors](#authors)
+
 ## DNS Seed Queries
 
-A client MAY issue queries using the `A`, `AAAA`, or `SRV` query types, specifying conditions for the desired results that the seed should return.
+A client MAY issue queries using the `A`, `AAAA`, or `SRV` query types,
+specifying conditions for the desired results the seed should return.
+
+Queries distinguish between _wildcard_ queries and _node_ queries, depending on
+whether the `l`-key is set or not.
 
 ### Query Semantics
 
-The conditions are key-value pairs with a single-letter key; the remainder of the key-value pair is the value.
+The conditions are key-value pairs: the key is a single-letter, while the
+remainder of the key-value pair is the value.
 The following key-value pairs MUST be supported by a DNS seed:
 
- - `r`: realm byte, used to specify what realm the returned nodes must support (default value: 0, Bitcoin)
- - `a`: address types, used to specify what address types should be returned for `SRV` queries. This is a bitfield that uses the types from [BOLT #7](07-routing-gossip.md) as bit index. This condition MAY only be used for `SRV` queries. (default value: 6, i.e. `2 || 4`, since bit 1 and bit 2 are set for IPv4 and IPv6, respectively)
- - `l`: `node_id`, the bech32-encoded `node_id` of a specific node, used to ask for a single node instead of a random selection. (default: null)
- - `n`: the number of desired reply records (default: 25)
+ - `r`: realm byte
+   - used to specify what realm the returned nodes must support
+   - default value: 0 (Bitcoin)
+ - `a`: address types
+   - used to specify what address types should be returned for `SRV` queries
+   - a bitfield that uses the types from [BOLT #7](07-routing-gossip.md) as bit
+   index
+   - MAY only be used for `SRV` queries
+   - default value: 6 (i.e. `2 || 4`, since bit 1 and bit 2 are set for IPv4 and
+     IPv6, respectively)
+ - `l`: `node_id`
+   - bech32-encoded `node_id` of a specific node
```

Comment:
Prefacing this with "a" will make it more consistent with the `r` description

Reply:
done